bindings/wasm/lib/gltf-node.ts

```diff
@@ -140,7 +160,27 @@ export class NonManifoldGLTFNode extends BaseGLTFNode {
    */
   makeManifold(): Manifold {
     return gltfNodeToManifold(this.document, this.node);
-  };
+  }
+
+  /**
+   * Does this node contain non-manifold geometry?
+   */
+  isNonManifold() {
```

Comment:
nit pick, but non-manifold means the topology has open edges. I think what you mean here is "not guaranteed manifold", or even "hasn't touched the Manifold library". It's already an overloaded term here. Maybe reverse the true/false and call it `fromManifold`?